tools/crm_resource.c

```diff
@@ -589,6 +625,40 @@ main(int argc, char **argv)
                     lrmd_api_delete(lrmd_conn);
                     crm_exit(exit_code);
 
+                } else if (safe_str_eq("class", longname)) {
+                    if (!(pcmk_get_ra_caps(optarg) & pcmk_ra_cap_params)) {
+                        fprintf(stderr, "Standard %s does not support parameters\n",
+                                optarg);
```

Comment:
Perhaps skip this message if --quiet is given. That's not terribly important, but we should try to respect --quiet when adding new messages. Also, maybe stdout rather than stderr?

I keep thinking we should do something useful for non-ocf standards, but figuring out what is probably not worth it at this stage.

Reply:
I was following the surrounding code there when it came to stderr vs. stdout, but no problem.  Changed.